game.js

```diff
@@ -27,52 +27,85 @@ function getRandom(array) {
   return array[idx];
 }
 
-/**
- * Adds and removes randomly selected classes to randomly selected trumps
- *
- */
-function randTrump() {
-  // Available classes for trump position
-  const positions = [
+// Available classes for trump position
+const positions = [
     'animateTop',
     'animateLeft',
     'animateRight'
-  ];
+];
 
-  // Available classes for trumps
-  const trumps = [
+// Available classes for trumps
+const trumps = [
     '.trump01',
     '.trump02',
     '.trump03',
     '.trump04',
     '.trump05',
     '.trump06'
-  ];
-  setInterval(() => {
-    let randomTrump = document.querySelector(
-      getRandom(trumps)
-    );
+];
+
+// Starts random trump appearance
+let gameActive = setInterval(() => {
+    let randomTrump = document.querySelector(getRandom(trumps))
     let randPosition = getRandom(positions);
     randomTrump.classList.add('animate', randPosition);
     setTimeout(() => {
-      randomTrump.classList.remove('animate', randPosition);
-    }, 1500);
-    console.log(randomTrump);
-  }, randTime(2000, 4000));
-}
-
-randTrump();
+            randomTrump.classList.remove('animate', randPosition);
+        },1500)
+}, randTime(2000, 4000));
 
 let trumpsDiv = document.querySelectorAll('.trump');
 let score = 0;
 
-//Wrap in if statement to check if trump is popped up first
-trumpsDiv.forEach((trump) =>
+// Animates on click and adds to score
+trumpsDiv.forEach(trump =>
   trump.addEventListener('click', () => {
     if (trump.classList.contains('animate')) {
       trump.classList.remove('animate');
       score += 1;
       document.querySelector('.score').textContent = score;
     }
-  })
-);
+}))
+
+const modalContainer = document.querySelector(".modalContainer")
+const modal = document.querySelector(".modal")
+const gameOverTitle = document.querySelector(".gameOverTitle h1")
+const gameOverMess = document.querySelector(".gameOverMess h3")
+let gif = document.querySelector(".gif img")
+let title
+let message
+
+// Starts countdown and displays game over popup at end
+let timer = document.querySelector('.timer').textContent;
+let countdown = setInterval(() => {
+    timer--;
+    document.querySelector('.timer').textContent = timer;
+    if (timer <= 0) {
+        clearInterval(countdown);
+        clearInterval(gameActive);
+        modalContainer.style.visibility = 'visible';
+        switch (true) {
```

Comment:
Why pass in true here? This is usually done as workaround to allow a switch to check more than a single variable, but in this switch every case is checking score. 

Pass in the score instead and then your cases can be simplified to just `case 0:`

Reply:
@ashleycoles but we're using an expression for the 3rd case - I thought that's when you needed true?

Reply:
Quite right!

Reply:
I was merely testing you 👀